README.md

```diff
@@ -33,3 +42,11 @@ You can start editing the page by modifying `pages/index.tsx`. The page auto-upd
 [API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.ts`.
 
 The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
+
+# TODO
+
+1. Search this repo for 'TODO' to find things that need to be done.
+1. Also search for 'eslint-disable' and fix those.
+1. Add unit tests.
+1. Add end-to-end tests.
+1. Add branch protection rules for `main` and `develop`.
```

Comment:
Lets remove this, we can add TODO in our backlog

Reply:
@nearken yep, sounds fine with me